README.md

```diff
@@ -1 +1,31 @@
-# ratify-cli
\ No newline at end of file
+# ratify-cli
+
+> [!WARNING]
+> This project is currently under initial development. Commands may change incompatibly.
+
+`ratify-cli` is the v2 version of Ratify's command-line tool for managing and verifying reference artifacts
+
+## Versioning
+
+This CLI follows [Semantic Versioning](https://semver.org/). Breaking changes are reserved for major releases, while minor and patch releases remain fully backward compatible.
+
+## v2: under development
+[![Build Status](https://github.com/ratify-project/ratify-cli/actions/workflows/build.yml/badge.svg?event=push&branch=main)](https://github.com/ratify-project/ratify-cli/actions/workflows/build.yml?query=event%3Apush+branch%3Amain)
+[![Codecov](https://codecov.io/gh/ratify-project/ratify-cli/branch/main/graph/badge.svg)](https://codecov.io/gh/ratify-project/ratify-cli)
+
+The version 2 of Ratify is under development.
+
+## v1: stable
+As v1 has been stable for a while, it is still available in the [Ratify repository](https://github.com/ratify-project/ratify)
```

Comment:
Let's group them in `Project Status` just like `oras-go`.

Reply:
Updated.